db/db.js

```diff
@@ -7,4 +7,15 @@ exports.checkArticleExists = (article_id) => {
         }
         return res.rows[0]
     })
-}
\ No newline at end of file
+}
```

Comment:
This function looks very similar to the checkArticleExists function above. You could refactor both into one, slightly more dynamic, function to check if data exists. Have a look at the notes put in the seminar slack channel on Tuesday 😄